packages/clerk-js/src/ui/primitives/Button.tsx

```diff
@@ -8,7 +8,7 @@ import { Spinner } from './Spinner';
 
 const vars = createCssVariables('accent', 'accentDark', 'accentDarker', 'accentLighter', 'accentLightest', 'border');
 
-const { applyVariants, filterProps } = createVariants(theme => {
+const { applyVariants, filterProps } = createVariants((theme, props) => {
```

Comment:
Do we use these props?

Reply:
Right, we don't anymore. That got away. Thanks.